src/components/AddVac/index.tsx

```diff
@@ -1,27 +1,69 @@
 import React from 'react';
+import React, { useState } from 'react';
+import { API } from '../../constants/intex';
+import axios from 'axios';
 
 const AddVac = () => {
+    const [nameVacancy, setNameVacancy] = useState("");
+    const [salaryVacancy, setSalarylVacancy] = useState("");
+    const [priceFrom, setPriceFrom] = useState("");
+    const [nameCompany, setNameCompany] = useState("");
+    const [priceTo, setPriceTo] = useState("");
+    const [address, setAddress] = useState("");
+    const [phone, setPhone] = useState("");
+    const [email, setEmail] = useState("");
+    const handleAddVacancy = async (e: React.FormEvent) => {
+        e.preventDefault();
+        try {
+            const response = await axios.post(`${API}/jobs`, {
+                organization_name: nameCompany,
+                price_from: Number(priceFrom),
+                price_to: Number(priceTo),
+                currency: 'currency',
+                position: nameVacancy,
+                city: address,
+                salary: salaryVacancy,
+                phone: phone,
+                type: 'type',
+                id: Math.random(),
+                slug: '',
+                created_at: '',
+                updated_at: '',
+                is_archived: true,
+                gradient: 0,
+                workday: '',
+                organization_icon: '',
+                organization_icon_formats: [null]
```

Comment:
useState'terdi object string kilip handle kil

Refer to this code.
https://github.com/TinatinNasirdinovna/dev_kg/pull/22/files

Reply:
Kiyinki PR'da kilip kelseng bolot.